houston/ardu/connection.py

```diff
@@ -79,6 +79,7 @@ def __init__(self,
                                timeout=timeout,
                                raise_exception=True)
         self.__conn.wait_ready('autopilot_version',
+                               'home_location',
```

Comment:
This fixes the issue? Do you mind adding a small comment above this line that describes the fix? (So that we don't forget why we added this in the future :slightly_smiling_face:.)